src/main/webapp/app/course/learning-paths/learning-path-graph/learning-path-legend.component.html

```diff
@@ -1,12 +1,4 @@
 <div class="container legend-wrapper">
-    <div class="row" ngbTooltip="{{ 'artemisApp.learningPath.graph.legend.competencyStart.tooltip' | artemisTranslate }}">
```

Comment:
Just a question: Why were competency starts completely removed? I don't quite understand that.

Reply:
They're replaced by the competency rings.